cmd/init_test.go

```diff
@@ -11,11 +11,11 @@ import (
 )
 
 func TestInit(t *testing.T) {
-	t.Run("Initialising program succeeds", testInitialisingProgramSucceeds)
-	t.Run("Forcing program initialisation with force succeeds", testForcingProgramInitialisationSucceeds)
+	t.Run("Initialising software succeeds", testInitialisingSoftwareSucceeds)
+	t.Run("Forcing software initialisation with force succeeds", testForcingSoftwareInitialisationSucceeds)
```

Comment:
"Forcing software initialisation with force succeeds" reads a bit funny - could "forcing" be replaced with a more useful word that helps define the "force" command?

Reply:
Right, what about `Forcing software initialisation succeeds` ?